website/docs/lsp.md

```diff
@@ -154,6 +154,13 @@ A short list of useful LSP-related command line flags:
   diagnostics it sends to clients at 1,000 diagnostics, but this can be changed
   (set it to `0` to remove the cap).
 
+- `--cache=dir=...`
+
+  When an edit causes Sorbet to retypecheck the whole workspace, supplying a
```

Comment:
Let's link the "retypecheck the whole workspace" to https://sorbet.org/docs/server-status#why-do-some-of-my-edits-make-sorbet-go-back-to-typechecking

Reply:
Similarly, we need to update those docs to mention that `Indexing` can happen on the slow path now, not just on startup:

https://sorbet.org/docs/server-status#indexing-files